data/vibe/data/serialization.d

```diff
@@ -475,7 +475,10 @@ private template serializeValueImpl(Serializer, alias Policy) {
 			ser.serializeValue!(CustomType, ATTRIBUTES)(value.toRepresentation());
 		} else static if (isISOExtStringSerializable!TU) {
 			ser.serializeValue!(string, ATTRIBUTES)(value.toISOExtString());
-		} else static if (isStringSerializable!TU) {
+		} else static if (ObjectSupportSinkType!TU && SerializerSupportSinkType!(Serializer, TU)) {
```

Comment:
ObjectSupportSinkType is not technically needed as SerializerSupportSinkType implicitly checks for the same